rc/core/c-family.kak

```diff
@@ -145,7 +145,7 @@ def -hidden _c-family-insert-on-newline %[ eval -draft %[
 }
 
 # c specific
-addhl -group /c/code regex %{\bNULL\b|\b-?(0x[0-9a-fA-F]+|\d+)[fdiu]?|'((\\.)?|[^'\\])'} 0:value
+addhl -group /c/code regex %{\b-?(0x[0-9a-fA-F]+|\d+)[fdiu]?|'((\\.)?|[^'\\])'} 0:value
```

Comment:
Why remove NULL here ? The fact that its a macro is an implementation detail, NULL means the null pointer value in C.